src/DynamoCoreWpf/Views/HomePage/HomePage.xaml.cs

```diff
@@ -0,0 +1,572 @@
+
+            var assembly = Assembly.GetExecutingAssembly();
+
+            using (Stream stream = assembly.GetManifestResourceStream(htmlEmbeddedFile))
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                htmlString = reader.ReadToEnd();
+            }
+
+            using (Stream stream = assembly.GetManifestResourceStream(jsEmbeddedFile))
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                var jsString = reader.ReadToEnd();  
+                jsonString = jsString;
+            }
+
+            // Copy the embedded resource to a temporary folder
+            using (var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream(fontStylePath))
+            {
+                if (stream != null)
+                {
+                    var fontData = new byte[stream.Length];
+                    stream.Read(fontData, 0, fontData.Length);
+
+                    // Create the temporary folder if it doesn't exist
+                    Directory.CreateDirectory(virtualFolderPath);
+
+                    // Write the font file to the temporary folder
+                    fontFilePath = Path.Combine(virtualFolderPath, "ArtifaktElement-Regular.woff");
+                    File.WriteAllBytes(fontFilePath, fontData);
+                }
+            }
+
+            // Set up virtual host name to folder mapping
+            dynWebView.CoreWebView2.SetVirtualHostNameToFolderMapping(virtualFolderName, virtualFolderPath, CoreWebView2HostResourceAccessKind.DenyCors);
+
+            htmlString = htmlString.Replace("mainJs", jsonString);
+
+            try
+            {
+                dynWebView.NavigateToString(htmlString);
+            }
+            catch (Exception ex)
+            {
+                Debug.WriteLine(ex.Message);
+            }
+
+            // Exposing commands to the React front-end
+            dynWebView.CoreWebView2.AddHostObjectToScript("scriptObject",
+               new ScriptHomeObject(RequestOpenFile,
+                                    RequestNewWorkspace,
+                                    RequestOpenWorkspace,
+                                    RequestNewCustomNodeWorkspace,
+                                    RequestApplicationLoaded,
+                                    RequestShowGuidedTour,
+                                    RequestShowSampleFilesInFolder,
+                                    RequestShowBackupFilesInFolder,
+                                    RequestShowTemplate));
+        }
+
+        internal async void LoadingDone()   
+        {
+            SendGuidesData();
+
+            if (startPage == null) { return; }
+
+            SendSamplesData();
+
+            var recentFiles = startPage.RecentFiles;
+            if (recentFiles == null || !recentFiles.Any()) { return; }
+
+            LoadGraphs(recentFiles);
+
+            var userLocale = "en";
```

Comment:
This user locale should be dynamic right?

Reply:
Yes, of course, thank you for spotting this out - replaced with the current culture and added 'en-US' locale to the webapp.